mule-extensions-api/src/main/java/org/mule/runtime/extension/internal/loader/ExtensionModelFactory.java

```diff
@@ -213,6 +214,7 @@ public ExtensionModelFactory() {
                                                        new OperationModelValidator(),
                                                        new FunctionModelValidator(),
                                                        new ParameterModelValidator(),
+                                                       new ParameterPluralNameModelValidator(),
```

Comment:
putting this at this level will break RestConnect

Reply:
Moved it to the `DefaultModelLoaderDelegate` (**not** used by XML SDK but by everyone else).